ansible/roles/test/tasks/simple-fib.yml

```diff
@@ -6,7 +6,7 @@
 
 - debug : msg="Start FIB Test"
 
-- set_fact: mtu=9114
+- set_fact: mtu=1500
```

Comment:
This is a configurable parameter. Can we have this default to 9100 and in your test modify to 1500?. Prefer to keep the default as 9100.

Reply:
Sure. It's a better option. Then I will close this PR.